tests/test_spidermon_signal_connect.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+import pytest
+from scrapy import signals
+from scrapy.crawler import CrawlerRunner
+from scrapy.spiders import Spider
+
+from spidermon.contrib.scrapy.extensions import Spidermon
+
+
+def get_crawler(settings_dict=None):
+    runner = CrawlerRunner(settings_dict)
+    return runner.create_crawler(Spider)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 4), reason="requires python3.4 or higher")
+def test_spider_opened_connect_signal():
+    from unittest import mock
+
+    with mock.patch.object(Spidermon, "spider_opened") as spider_opened_method:
```

Comment:
If you use pytest-mock you can simplify it as (and avoid import):

    spider_opened_method = mocker.patch.object(Spidermon, "spider_opened")

another example https://github.com/scrapinghub/gatf/blob/2e7208228d8daebdb23e84c88483f16b2cbfb487/tests/test_gatf.py#L50

Also I always use `autospec`, since without it you can use imaginable methods and not even know the test doesn't work